pkg/skaffold/build/cache/retrieve.go

```diff
@@ -62,7 +63,10 @@ func (c *cache) Build(ctx context.Context, out io.Writer, tags tag.ImageTags, ar
 		result := results[i]
 		switch result := result.(type) {
 		case failed:
-			return nil, errors.Wrap(result.err, "checking cache")
+			logrus.Warnf("error checking cache, caching may not work as expected: %v", result.err)
+			color.Red.Fprintln(out, "Error checking cache. Rebuilding.")
```

Comment:
Maybe not red just orange? 
```suggestion
			color.Orange.Fprintln(out, "Error checking cache. Rebuilding.")
```

Reply:
We use red when we can't [find](https://github.com/GoogleContainerTools/skaffold/blob/master/pkg/skaffold/build/cache/retrieve.go#L68) the image and rebuild, so I figured it would look better to do the same here too. WDYT?

Reply:
Since caching is enabled by default, and it doesn't result in an error, red doesn't seem to be warranted.

Reply:
SGTM